### PR TITLE
DT POM EntityBoardTest  + ignored test 

### PR DESCRIPTION
**Pull request by 4stuntman4**
https://trello.com/c/xkMQilrs/518-editpage-pom-entityboardtest

https://trello.com/c/DS2qfmKS/529-marketplaceinstancetest  ignored

### CODE REVIEW

File: src/test/java/EntityBoardTest.java [hunk 1]
```diff
@@ -26,51 +21,57 @@
...
-    private static final LocalDate TODAY = LocalDate.now();
-    private static final String OUTPUT = TODAY.toString();
-    private static final String[] ARR_OF_DATA = OUTPUT.split("-", 3);
-    private static final String CURRENT_YEAR = ARR_OF_DATA[0];
-    private static final String CURRENT_MONTH = ARR_OF_DATA[1];
-    Random generator = new Random();
-    private final String RANDOM_DAY = String.format("%02d", generator.nextInt(27) + 1);
+    String dateForValidation;
+    String dateTimeForValidation;
+    String time;
```
Comment (Stan575) [inline src/test/java/EntityBoardTest.java:26]: you can make these private

**Review by Stan575 — APPROVED** [review]
Great job on calendar!

File: src/test/java/model/CalendarPage.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import runner.ProjectUtils;
+
+import java.time.LocalDate;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class CalendarPage extends MainPage{
```
[the remark this answers is not in the document]
Comment (4stuntman4) [reply src/test/java/model/CalendarPage.java]: done

File: src/test/java/EntityBoardTest.java (hunk 1 above)
Comment (4stuntman4) [reply src/test/java/EntityBoardTest.java]: done

**Review by GalinaRuban — APPROVED** [review]

**Pull request merged**